### PR TITLE
Add aks-engine v1.15 conformance results

### PR DESCRIPTION
**Pull request by lachie83**

### CODE REVIEW

File: v1.15/aks-engine/PRODUCT.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+vendor: Microsoft
+name: Azure aks-engine
+version: v0.37.1
```
Comment (CecileRobertMichon) [inline v1.15/aks-engine/PRODUCT.yaml]: should be v0.37.2

**Comment by dankohn** [comment]
An hour delay!?!
🐌

**Comment by lachie83** [comment]
Sorry. We'll try to up our game next time.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**